diff --git a/Client/Models/AppendMediaUploadResponseData.cs b/Client/Models/AppendMediaUploadResponseData.cs
index 21d87b9..e2c4719 100644
--- a/Client/Models/AppendMediaUploadResponseData.cs
+++ b/Client/Models/AppendMediaUploadResponseData.cs
@@ -41,7 +41,8 @@ namespace XbyOpenApi.Core.Client.Models
     {
       return new Dictionary<string, Action<IParseNode>>
             {
-                { "expires_at", n => { ExpiresAt = n.GetIntValue(); } },
+                //{ "expires_at", n => { ExpiresAt = n.GetIntValue(); } },
+                { "expires_after_secs", n => { ExpiresAt = n.GetIntValue(); } },
             };
     }
     /// <summary>
@@ -51,7 +52,8 @@ namespace XbyOpenApi.Core.Client.Models
     public virtual void Serialize(ISerializationWriter writer)
     {
       if (ReferenceEquals(writer, null)) throw new ArgumentNullException(nameof(writer));
-      writer.WriteIntValue("expires_at", ExpiresAt);
+      //writer.WriteIntValue("expires_at", ExpiresAt);
+      writer.WriteIntValue("expires_after_secs", ExpiresAt);
       writer.WriteAdditionalData(AdditionalData);
     }
   }
